schema/bom-1.7.xsd

```diff
@@ -7387,28 +7887,28 @@ limitations under the License.
                             </xs:annotation>
                             <xs:complexType>
                                 <xs:sequence>
-                                    <xs:element name="encr" type="bom:refType" minOccurs="0" maxOccurs="unbounded">
+                                    <xs:element name="encr" type="bom:ikeV2Enc" minOccurs="0" maxOccurs="unbounded">
                                         <xs:annotation>
                                             <xs:documentation>
                                                 Transform Type 1: encryption algorithms
                                             </xs:documentation>
                                         </xs:annotation>
                                     </xs:element>
-                                    <xs:element name="prf" type="bom:refType" minOccurs="0" maxOccurs="unbounded">
+                                    <xs:element name="prf" type="bom:ikeV2Prf" minOccurs="0" maxOccurs="unbounded">
                                         <xs:annotation>
                                             <xs:documentation>
                                                 Transform Type 2: pseudorandom functions
                                             </xs:documentation>
                                         </xs:annotation>
                                     </xs:element>
-                                    <xs:element name="integ" type="bom:refType" minOccurs="0" maxOccurs="unbounded">
+                                    <xs:element name="integ" type="bom:ikeV2Integ" minOccurs="0" maxOccurs="unbounded">
                                         <xs:annotation>
                                             <xs:documentation>
                                                 Transform Type 3: integrity algorithms
                                             </xs:documentation>
                                         </xs:annotation>
                                     </xs:element>
-                                    <xs:element name="ke" type="bom:refType" minOccurs="0" maxOccurs="unbounded">
```

Comment:
Incompatible change of types, from simple string to complex type.

:x: this is a breaking change

Reply:
Same.